src/gui/Main.java

```diff
@@ -11,19 +13,8 @@
 	@Override
 	public void start(Stage primaryStage) {
 		//primaryStage.setMaximized(true);
-		Utils.createWindow(primaryStage, Main.this, "./fxml/Home.fxml", "TasksFX", null, "./css/home.css");
-
-
-		/*primaryStage.showingProperty().addListener(new ChangeListener<Boolean>() {
-
-			public void changed(ObservableValue<? extends Boolean> observable, Boolean oldValue, Boolean newValue) {
-				// TODO Auto-generated method stub
-				if (newValue) {
-
-				}
-			}
+		Utils.createWindow(primaryStage, Main.this, "./fxml/Home.fxml", "TasksFX", null, "./css/Home.css", "resources.i18n.Home");
```

Comment:
The Home CSS and the resource should be lower case:
`./css/Home.css` -> `./css/home.css`
`resources.i18n.Home` -> `resources.i18n.home`